tests/integration/test_subordinate_charms.py

```diff
@@ -37,11 +38,29 @@ async def test_ubuntu_pro(ops_test, mysql_charm_series, github_secrets):
     await ops_test.model.relate(
         f"{DATABASE_APP_NAME}:database", f"{APPLICATION_APP_NAME}:database"
     )
-    await ops_test.model.relate(f"{DATABASE_APP_NAME}", f"{UBUNTU_PRO_APP_NAME}")
+    await ops_test.model.relate(DATABASE_APP_NAME, UBUNTU_PRO_APP_NAME)
     async with ops_test.fast_forward("60s"):
         await ops_test.model.wait_for_idle(
             apps=[DATABASE_APP_NAME, APPLICATION_APP_NAME, UBUNTU_PRO_APP_NAME],
             status="active",
             raise_on_blocked=True,
             timeout=TIMEOUT,
         )
+
+
+@pytest.mark.group(1)
+async def test_landscape_client(ops_test, github_secrets):
```

Comment:
is there something else to test on the landscape charm? How do we attest functionality?

Reply:
Goal: make sure subordinated landscape-client is not affecting principal data charm functionality.
No need to test landscape itself here.